GTMAppAuthSwift/Sources/KeychainConfiguration.swift

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+enum KeychainAttribute {
+  case dataProtectionKeychain
```

Comment:
The keychain attribute takes a boolean value.  Should we model that here for consistency?

Reply:
I'm assuming that creating this attribute means `true`. In other words, we don't need to know the value of the Boolean since if `false` is wanted, then devs simply don't need to create the attribute.

Reply:
Closing this per an offline conversation where we agreed that the `public` API with `.useDataProtectionKeychain` made things clear.